src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindUsageProperty.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.FindSymbols.FindReferences
+{
+    internal readonly struct FindUsageProperty
```

Comment:
odd name given the namespace.  maybe AdditionalProperty or just Property.

Reply:
This will be addressed in the followup PR: https://github.com/dotnet/roslyn/issues/37961